docs/index.rst

```diff
@@ -108,3 +108,11 @@ experience necessary):
 * `Try the development version <http://astropy.readthedocs.io/en/stable/development/workflow/get_devel_version.html>`_
 
 * `Developer Documentation <http://docs.astropy.org/en/latest/#developer-documentation>`_
+
+Citing Photutils
+================
+
+If you use Photutils, please consider citing the package via it's Zenodo record.
```

Comment:
Obviously, I can do this myself... but @larrybradley, in the interest of testing: can you see if *you* can make this edit?  I turned on the "allow edits from maintainers" option, which I *think* means you can do this, but I have yet to test it.  Now seems like a good opportunity!